### PR TITLE
Added option to ignore robots.txt

### PR DESCRIPTION
**Pull request by Gayoushie**
Added option to bypass robots.txt file, so that the tool can be used to team servers, where all urls are disallowed by robots.txt file.

### CODE REVIEW

**Comment by Hywan** [comment]
Hello,

Nice contribution, thanks! But what do you mean by “be used to team servers”?

**Comment by Gayoushie** [comment]
Hi. While trying to use the A11yMachine we came across this problem... we have a production site, on which it worked perfectly, but we wanted to test our other servers (which we call team servers): dev, QA, stage, etc. Our site goes through different stages before it can go live, and we would like to test for accessibility before things go to production. And all our servers, except for production, have robots.txt with all pages disallowed. I think this is a common practice with many companies. 

Hope I clarified your question. 

Thanks,
Gayane

File: README.md [hunk 1]
```diff
@@ -120,6 +120,7 @@ As a prelude, see the help:
     --http-auth-password <http_auth_password>  Password to authenticate all HTTP requests.
     --http-tls-disable                         Disable TLS/SSL when crawling or downloading pages.
     -V, --no-verbose                           Make the program silent.
+    --ignore-robots                            Ignore robots.txt file.
```
Comment (Hywan) [inline README.md]: I would call this option `--ignore-robots-txt`. Is it possible to rename it?

File: a11ym [hunk 2]
```diff
@@ -114,6 +114,10 @@ program
         '-V, --no-verbose',
         'Make the program silent.'
     )
+    .option(
+        '--ignore-robots',
```
Comment (Hywan) [inline a11ym]: Same here.

File: lib/a11ym.js [hunk 3]
```diff
@@ -58,7 +58,8 @@ var defaultOptions = {
     httpAuthUser    : undefined,
     httpAuthPassword: undefined,
     httpTlsDisable  : undefined,
-    verbose         : true
+    verbose         : true,
+    ignoreRobots    : false
```
Comment (Hywan) [inline lib/a11ym.js]: `ignoreRobotsTxt` then.

**Comment by Hywan** [comment]
@Gayoushie Yes it is!

**Comment by Gayoushie** [comment]
hi @Hywan, per your request i renamed the option to --ignore-robots-txt.

**Comment by Hywan** [comment]
Thanks! We will take a look at it as soon as possible!

File: README.md [hunk 4]
```diff
@@ -120,6 +120,7 @@ As a prelude, see the help:
     --http-auth-password <http_auth_password>  Password to authenticate all HTTP requests.
     --http-tls-disable                         Disable TLS/SSL when crawling or downloading pages.
     -V, --no-verbose                           Make the program silent.
+    --ignore-robots-txt                        Ignore robots.txt file.
```
Comment (patkar) [inline README.md]: `--ignore-robots` could be enough ;)

File: README.md (hunk 1 above)
Comment (patkar) [inline README.md]: But why? The file extension part is not an information you need in the option ;) `--ignore-robots` should be enough to understand it IMHO.

**Pull request merged**